migrations/env.py

```diff
@@ -36,7 +36,7 @@
 # my_important_option = config.get_main_option("my_important_option")
 # ... etc.
 
-url = make_database_url()
+url = conf.SQLALCHEMY_DATABASE_URI
```

Comment:
I believe I used this at one point because I'm having an interesting error now. The default db is falling back to qualitas_db which is fine. However, when i run migrations in my tests it's not picking up tests. This seems to by why I had defaulted to tests before. I'm having trouble trying to come up with a better way.

Reply:
yea, after banging my head on this for a while we're going to need to use `make_database_url()`. i tried monkeypatching but then pytest complained monkeypatch is a `session="function"` fixture and my app fixture is session. At this time, there is no way for alembic to know what database to use. My workaround is to set the env var in my test framework and use the function. I'll need to look into using Flask-Alembic or Flask-Migrate to make this better. For now, it works and that's fine for me. =)

Reply:
pffttt. actually that still doesn't work. it seems for now defaulting to "tests" and overriding that for running the app is what has to happen ... for now.

Reply:
I may need to change the way my configuration is done. This setup was when I did deployments with ansible. I would template out the `instance/conf.py` file and have ansible fill it out based on the template vars. Now that I'm using heroku I probably need to change it up.